client/components/first-view/index.jsx

```diff
@@ -43,7 +43,9 @@ const FirstView = React.createClass( {
 	},
 
 	componentWillUnmount() {
-		this.updateDocumentStylesForHiddenFirstView();
+		process.nextTick( () => {
```

Comment:
> but in general I think we should favor built-in functions as opposed to shims

> if every time we could use a standard built-in function we instead used a slightly-semantically-better shim then Calypso would become bloated

I 100% agree, but in this case the bloat is already present. Not using it wouldn't make things any less bloated.

Thanks for the feedback and reviews!